resources/js/Pages/Auth/Register.vue

```diff
@@ -39,49 +39,53 @@
         <v-col cols="12">
           <v-text-field
             v-model="form.password"
+            :append-icon="show1 ? 'mdi-eye' : 'mdi-eye-off'"
+            :type="show1 ? 'text' : 'password'"
             name="password"
-            type="password"
-            label="Password"
+            :label="$t('auth.password')"
             hide-details="auto"
             autocomplete="new-password"
             :error-messages="errors['password']"
             outlined
             required
             autofocus
+            @click:append="show1 = !show1"
           />
         </v-col>
         <v-col cols="12">
           <v-text-field
             v-model="form.password_confirmation"
+            :append-icon="show2 ? 'mdi-eye' : 'mdi-eye-off'"
             name="password"
-            type="password"
-            label="Confirm Password"
+            :type="show2 ? 'text' : 'password'"
+            :label="$t('auth.passwordconfirmation')"
             hide-details="auto"
             autocomplete="new-password"
             outlined
             required
             autofocus
+            @click:append="show2 = !show2"
           />
         </v-col>
-        <v-col
+        <!-- <v-col
```

Comment:
if we don't need it. then it's best to remove it from the code also (with it's according translation keys)